common/src/main/java/tachyon/network/protocol/databuffer/DataBuffer.java

```diff
@@ -42,4 +42,11 @@
    * @return a read-only ByteBuffer representing the data
    */
   public abstract ByteBuffer getReadOnlyByteBuffer();
+
+  /**
+  * Release the underlying buffer of this DataBuffer if no longer needed.
+  *
+  * @return True if the underlying buffer is released or no need to release
+  */
+  public abstract boolean release();
```

Comment:
Should this just be a void method? I don't think the return boolean is helpful. Also, the Netty version doesn't return what the comment is describing.